### PR TITLE
Documenting conversion rules for operations involving PointVector.

### PR DESCRIPTION
**Pull request by rolanddenis**
# PR Description

Documentation of conversion rules when manipulating `PointVector` (see #1345).

# Checklist

- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [x] All continuous integration tests pass (Travis & appveyor)

### CODE REVIEW

File: src/DGtal/kernel/PointVector.h [hunk 1]
```diff
@@ -105,16 +105,36 @@ namespace DGtal
    * exact same behaviour.
    *
    *
-   * If TEuclideanRing is a Integer type (built-in integers,
-   * BigIntegers, ...), the "/" operator on Points corresponds to
-   * component by component Euclidean division.
+   * All operations involving @ref PointVector, and some of its methods,
+   * follows the classical arithmetic conversion rules.
```
Comment (dcoeurjo) [inline src/DGtal/kernel/PointVector.h]:
```suggestion
   * follow the classical arithmetic conversion rules.
```

File: src/DGtal/kernel/PointVector.h [hunk 2]
```diff
@@ -105,16 +105,36 @@ namespace DGtal
...
+   * This constraint can be dodged by using one of the conversion constructor
+   * (explicitly or along with a conversion functor) or equivalent methods.
+   *
+   * @see ArithmeticConversionTraits
+   * @see https://en.cppreference.com/w/cpp/language/operator_arithmetic#Conversions
    *
-   * If TEuclideanRing is a double, the "/" operator on Points
-   * correspond to the classical division on real numbers (x*1/x = 1).
    *
+   * For example, dividing a @ref PointVector with integer component by
```
Comment (dcoeurjo) [inline src/DGtal/kernel/PointVector.h]:
```suggestion
   * For example, dividing a @ref PointVector with integer components by
```

File: src/DGtal/kernel/PointVector.h [hunk 3]
```diff
@@ -105,16 +105,36 @@ namespace DGtal
...
+   * an integer will use the Euclidean division and returns a @ref PointVector
```
Comment (dcoeurjo) [inline src/DGtal/kernel/PointVector.h]:
```suggestion
   * an integer uses the Euclidean division and returns a @ref PointVector
```

File: src/DGtal/kernel/PointVector.h [hunk 4]
```diff
@@ -105,16 +105,36 @@ namespace DGtal
...
+   * with higher integer component type.
+   * On the other hand, dividing a @ref PointVector with integer component by
```
Comment (dcoeurjo) [inline src/DGtal/kernel/PointVector.h]:
```suggestion
   * On the other hand, dividing a @ref PointVector with integer components by
```

**Comment by rolanddenis** [comment]
Thanks!

**Comment by dcoeurjo** [comment]
Thanks to you !
Merging and releasing 🎉

**Pull request merged**